src/consumer.ts

```diff
@@ -239,7 +239,7 @@ export class Consumer extends EventEmitter {
     }
   }
 
-  private async terminateVisabilityTimeout(message: SQSMessage): Promise<PromiseResult<any, AWSError>> {
+  private async terminateVisibilityTimeout(message: SQSMessage): Promise<PromiseResult<any, AWSError>> {
```

Comment:
Hi @gregory, you cannot update the function name to terminateVisibilityTimeout as we have a private property defined with this name already.